bayes_opt/bayesian_optimization.py

```
@@ -442,3 +446,233 @@ def load_state(self, path: str | PathLike[str]) -> None:
+
+        # Override the default acquisition function to Expected Improvement if not specified
+        if acquisition_function is None:
+            self._acquisition_function = acquisition.ExpectedImprovement(
+                xi=0.01, random_state=self._random_state
+            )
+
+        # Remove the default GP regressor, as we'll use a Pyro-based GP
+        self._gp = None
+        self._mcmc_samples = None
+
+    def _define_gp_model(self, X: torch.Tensor, y: torch.Tensor) -> Callable:
```

Pylint (3.3.7)

[convention] Argument name "X" doesn't conform to snake_case naming style

(C0103)